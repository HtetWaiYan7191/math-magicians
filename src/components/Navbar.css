nav {
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 15px 20px;
    background-color:rgb(15, 23, 42);
    color: rgb(255, 255, 255);
    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
}

h1 {
    font-size: 48px;
    font-weight: 500;
    color:#22D3EE
}

nav ul {
    display: flex;
}

nav ul li {
    list-style: none;
    margin: 0px 10px;
}

nav ul li a {
    font-size: 20px;
    text-decoration: none;
    color: #EEEEEE;
}

.active {
    color: #22D3EE;
}

nav ul a:hover {
    color: #22D3EE;
}