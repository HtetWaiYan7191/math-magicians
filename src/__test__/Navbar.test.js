import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect'; // Import the jest-dom extension
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar', () => {
  test('should render a component', () => {
    render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );
    expect(screen.getByText('Home')).toBeInTheDocument();
    expect(screen.getByText('Calculator')).toBeInTheDocument();
    expect(screen.getByText('Quote')).toBeInTheDocument();
  });
});
