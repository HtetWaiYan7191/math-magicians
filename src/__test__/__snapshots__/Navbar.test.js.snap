// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render a component 1`] = `null`;
